.drone.yml

```diff
@@ -5714,182 +5186,8741 @@ volumes:
+  - apk add --no-cache aws-cli
+  - TIMESTAMP=$(date -d @"$DRONE_BUILD_CREATED" '+%Y%m%d%H%M')
+  - aws ecr-public get-login-password --region=us-east-1 | docker login -u="AWS" --password-stdin
+    public.ecr.aws
+  - docker pull "drone-docker-registry:5000/teleport-operator-v10-arm64"
+  - docker tag "drone-docker-registry:5000/teleport-operator-v10-arm64" "public.ecr.aws/u8j2q1d9/teleport-operator:10-arm64"
+  - docker push "public.ecr.aws/u8j2q1d9/teleport-operator:10-arm64"
+  - docker logout "public.ecr.aws"
+  environment:
+    AWS_ACCESS_KEY_ID:
+      from_secret: TEST_PRODUCTION_TELEPORT_DRONE_USER_ECR_KEY
+    AWS_SECRET_ACCESS_KEY:
+      from_secret: TEST_PRODUCTION_TELEPORT_DRONE_USER_ECR_SECRET
+  volumes:
+  - name: dockersock
+    path: /var/run
+  depends_on:
+  - Build teleport-operator image "teleport-operator-v10-arm64"
+- name: Create manifest and push "public.ecr.aws/u8j2q1d9/teleport-operator:10" to
+    ECR - production
+  image: docker
+  commands:
+  - apk add --no-cache aws-cli
+  - TIMESTAMP=$(date -d @"$DRONE_BUILD_CREATED" '+%Y%m%d%H%M')
+  - aws ecr-public get-login-password --region=us-east-1 | docker login -u="AWS" --password-stdin
+    public.ecr.aws
+  - docker manifest create "public.ecr.aws/u8j2q1d9/teleport-operator:10" --amend
+    "public.ecr.aws/u8j2q1d9/teleport-operator:10-amd64" --amend "public.ecr.aws/u8j2q1d9/teleport-operator:10-arm"
+    --amend "public.ecr.aws/u8j2q1d9/teleport-operator:10-arm64"
+  - docker manifest push "public.ecr.aws/u8j2q1d9/teleport-operator:10"
+  - docker logout "public.ecr.aws"
+  environment:
+    AWS_ACCESS_KEY_ID:
+      from_secret: TEST_PRODUCTION_TELEPORT_DRONE_USER_ECR_KEY
+    AWS_SECRET_ACCESS_KEY:
+      from_secret: TEST_PRODUCTION_TELEPORT_DRONE_USER_ECR_SECRET
+  volumes:
+  - name: dockersock
+    path: /var/run
+  depends_on:
+  - Tag and push "public.ecr.aws/u8j2q1d9/teleport-operator:10-amd64" to ECR - production
+  - Tag and push "public.ecr.aws/u8j2q1d9/teleport-operator:10-arm" to ECR - production
+  - Tag and push "public.ecr.aws/u8j2q1d9/teleport-operator:10-arm64" to ECR - production
+services:
+- name: Start Docker
+  image: docker:dind
+  privileged: true
+  volumes:
+  - name: dockersock
+    path: /var/run
+- name: drone-docker-registry
+  image: registry:2
+  privileged: false
+  volumes: []
+volumes:
+- name: dockersock
+  temp: {}
+
+---
+################################################
+# Generated using dronegen, do not edit by hand!
+# Use 'make dronegen' to update.
+# Generated at dronegen/container_images.go:231
+################################################
+
+kind: pipeline
+type: kubernetes
+name: teleport-container-images-current-version-cron
+environment:
+  DEBIAN_FRONTEND: noninteractive
+trigger:
+  cron:
+    include:
+    - teleport-container-images-cron
+  repo:
+    include:
+    - gravitational/teleport
+workspace:
+  path: /go
+clone:
+  disable: true
+steps:
+- name: Find the latest available semver for v10
```

Comment:
The `cron` pipelines currently only run within the master branch. You are right that we could create separate cron pipelines that run within `branch/v*` instead.

Reply:
Logan is right. In short drone/our cron triggers are only setup to run in the context of the master branch. As a result we have to tell drone what the latest and last two releases are so that we can update the images. This is how the current image cron piplines work and I didn't really want to change it as a part of this PR.